components/core/Profile.js

```diff
@@ -175,15 +175,55 @@ const STYLES_FLEX = css`
   }
 `;
 
+const STYLES_EXPLORE = css`
+  padding-top: 8px;
+  margin: 200px auto 24px auto;
+  font-size: ${Constants.typescale.lvl2};
+  font-family: ${Constants.font.medium};
+  font-weight: 400;
+  color: ${Constants.system.black};
+`;
+
 export default class Profile extends React.Component {
+  state = {
+    exploreSlates: [],
+  };
+
+  componentDidMount = async () => {
+    let exploreSlates = [];
+    let response1 = await Actions.getSlateById({ id: "857ad84d-7eff-4861-a988-65c84b62fc23" });
```

Comment:
Might be worthwhile to spend some time to learn how to write a query such as

`Actions.getSlatesByIds`

Since you have less deadline pressure. and it would be useful for everyone, @martinalong if you could help.